tests/Logging.XUnit.Tests/XUnitLoggerTests.cs

```diff
@@ -473,6 +479,140 @@ public static void XUnitLogger_Log_Logs_Message_If_Scopes_Included_And_There_Are
             mock.Verify((p) => p.WriteLine(expected), Times.Once());
         }
 
+        [Fact]
+        public static void XUnitLogger_Log_Logs_Message_If_Scopes_Included_And_There_Is_Scope_Of_KeyValuePair()
+        {
+            // Arrange
+            var mock = new Mock<ITestOutputHelper>();
+
+            string name = "MyName";
+            var outputHelper = mock.Object;
+
+            var options = new XUnitLoggerOptions()
+            {
+                Filter = FilterTrue,
+                IncludeScopes = true,
+            };
+
+            var logger = new XUnitLogger(name, outputHelper, options)
+            {
+                Clock = StaticClock,
+            };
+
+            string expected = string.Join(
+                Environment.NewLine,
+                new[] { "[2018-08-19 16:12:16Z] info: MyName[0]", "      => ScopeKey: ScopeValue", "      Message|False|False" });
```

Comment:
Might as well have this one match the others.

```suggestion
                "[2018-08-19 16:12:16Z] info: MyName[0]",
                "      => ScopeKey: ScopeValue",
                "      Message|False|False");
```